server/db/db.js

```diff
@@ -40,6 +40,7 @@ class DBClient {
       dbPassword: process.env.DB_PASSWORD || 'development',
       dbName: process.env.DB_NAME || 'development',
       dbTLSEnabled: process.env.DB_AWS_TLS_ENABLED === 'true',
+      dbTLSDisableCert: process.env.DB_AWS_TLS_DISABLE_CERTIFICATE === 'true',
```

Comment:
Can you explain the purpose of this?

I think I know the answer but I don't want to assume and not ask :)

Reply:
Since we have a tunnel to the db port (connecting through localhost), using a certificate will result in a timeout error.

Reply:
👌